external/settings.php

```diff
@@ -7,6 +7,12 @@
 
 $tmpl = new OCP\Template( 'external', 'settings');
 
-$tmpl->assign('images', glob(\OC_App::getAppPath('external') . '/img/*.*'));
+$images = glob(\OC_App::getAppPath('external') . '/img/*.*');
+$theme = \OC_Util::getTheme();
```

Comment:
@PVince81 I like that approach for now. It's compatible with existing versions but can be replaced when there's a replacement in the public api.

If the returned string is empty then the file_exists check should fail due to a non-existant path. It tests ok in 8.2.5.